Xaml Designer Improvements

Redth opened the pull request:
- WIP

Redth committed:
Add attributes to xaml controls

Redth committed:
Migrate design dll's to new designtools extensibility

Redth committed:
Update extensibility nuget version

Redth committed:
Rename font size converter, support isvalid with name or double

Redth committed:
Add exclusive setting to named converter base

Redth committed:
Use Colors (plural) for looking at fields/properties for named colors

Redth committed:
Fix rename of font converter

Redth committed:
Adding more detection for valid colors to designtools converter

Redth committed:
Add controls' color converter implementation as type typeconverter attr

Redth committed:
Update Graphics version

Redth committed:
Use type name comparison

Redth committed:
Only use designtools color converter

mgoertz-msft commented on src/Controls/src/Core.Design/KnownValuesDesignTypeConverter.cs:
```diff
@@ -13,13 +13,13 @@ public abstract class KnownValuesDesignTypeConverter : TypeConverter
 		// Without this the values won't show up as hints
 		public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
-			=> sourceType == typeof(string) || base.CanConvertFrom(context, sourceType);
+			=> sourceType.FullName == typeof(string).FullName || base.CanConvertFrom(context, sourceType);
```
How come sourceType == typeof(string) doesn't work? That's what we do in our code.

Redth replied:
Was grasping at straws. It seems fine and I can likely revert this to do the same.

Redth committed:
Improve color validation for other strings

Redth committed:
Rename color converter

Redth committed:
Code cleanup

Redth committed:
Clean up type comparison

Redth committed:
Fix obsoleted method use

Redth committed:
Fix another obsoleted method use

Redth committed:
Fix obsolete color method usage

Redth committed:
Only pack designtools dlls on windows

Redth committed:
Fix nuget inclusion path

Redth committed:
Include design pdb

Nuget gets a bit complainy about not having symbols in a package when the rest are there for other assemblies.

mgoertz-msft commented:
Great, carry on then... :-)

Commit:
Try and fix duplicate file added pack error
